tests/wicked/startandstop/sut/t13_bridge_ifreload_all.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Bridge - ifreload with bond interfaces
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+
+sub run {
+    my ($self, $ctx) = @_;
+    $self->get_from_data('wicked/ifreload-3.sh', '/tmp/ifreload-3.sh');
+    my $script_cmd = sprintf(q(bond_slaves='%s' sh /tmp/ifreload-3.sh), join(" ", $ctx->iface, $ctx->iface2));
+    my $output     = script_output($script_cmd . '; echo "==COLLECT_EXIT_CODE==$?=="');
+    my $result     = $output =~ m/==COLLECT_EXIT_CODE==0==/ ? 'ok' : 'fail';
+    $self->record_extra_test_result('ifreload-3', $output, result => $result);
+}
+
+sub record_extra_test_result {
+    my ($self, $title, $content, %args) = @_;
+    $args{result} //= 'failed';
+    $title =~ s/:/_/g;
+    my $filename     = $self->next_resultname('txt', $title);
+    my $extra_result = {
+        _source => 'parser',
+        result  => $args{result},
+        text    => $filename,
+        title   => $title,
+    };
+    $self->write_resultfile($filename, $content);
```

Comment:
@okurz  , @Martchus  - WDYT ?

Reply:
I think it is more a question of introducing something like 
```
$self->record_console_test_result('ifreload-3', $output, result => $result);
```
I think this is really useful for console based tests as it makes the output in openqa-ui looking quite nice.

I'm going to create a PR in os-autoinst, but let's not stop this PR with such look&feel thing, or?